context.go

```diff
@@ -75,6 +76,13 @@ func (c *Ctx) LoadProject(path string) (*Project, error) {
 		return nil, err
 	}
 
+	// the path may lie within a symlinked directory, resolve that
```

Comment:
nit: I'm learning that go/stdlib coding standards generally look for complete sentences w/punctuation and capitalization in comments. It's also OK to have a just a couple explanatory words - but this sorta thing (a complete sentence, but lacking proper in these details) is a no-no.

(Something like that, @rakyll ? 😄 )